coderd/database/migrations/000198_org_provisioners.up.sql

```diff
@@ -0,0 +1,14 @@
+-- At the time of this migration, only 1 org is expected in a deployment.
+-- In the future when multi-org is more common, there might be a use case
+-- to allow a provisioner to be associated with multiple orgs.
+ALTER TABLE provisioner_daemons
+	ADD COLUMN organization_id UUID REFERENCES organizations(id) ON DELETE CASCADE;
+
+UPDATE
+	provisioner_daemons
+SET
+	-- Default to the first org
+	organization_id = (SELECT id FROM organizations ORDER BY organizations.created_at ASC LIMIT 1 );
```

Comment:
it's an odd case, but this will fail if there is a provisioner daemon but no default org, which could happen if you start coderd on an older version (which creates in-memory provisioner daemons) but don't create the first user, and then migrate.

Reply:
Yes, we need this PR in first: https://github.com/coder/coder/pull/12412